singer_sdk/helpers/capabilities.py

```diff
@@ -50,6 +50,34 @@
         description="The max depth to flatten schemas.",
     ),
 ).to_dict()
+BATCH_CONFIG = PropertiesList(
+    Property(
+        "batch_config",
+        description="",
+        wrapped=ObjectType(
+            Property(
+                "encoding",
+                description="Specifies the format and compression of the batch files.",
+                wrapped=ObjectType(
+                    Property("format", StringType, allowed_values=["jsonl"]),
+                    Property(
+                        "compression",
+                        StringType,
+                        allowed_values=["gzip", "none"],
```

Comment:
I think omitting "compression" is the same as "none". While "none" isn't strictly handled, the condition I found checks for "gzip" and is a no-op otherwise. @edgarrmondragon - Thoughts?

Reply:
Yeah, this works fine and is explicit 👍